client/src/pages/Signin.jsx

```diff
@@ -1,8 +1,39 @@
-import React from "react";
-import Header from "../components/header/index";
-import Footer from "../components/footer/index";
+import React, { useState } from "react";
+import Header from "../components/Header/index";
+import Footer from "../components/Footer/index";
 
 export default function Signin() {
```

Comment:
tiny thing:
al ser 2 palabras y si usamos la convencion camelCase it should be `SignIn`